src/Network/Haskoin/Store/Data.hs

```diff
@@ -332,6 +368,13 @@ instance BinSerial Unspent where
         putWord64be v
         put $ B.Short.fromShort s
```

Comment:
We don't need to convert the `ByteString` to a `ShortByteString` for serialization, as the `Serialize` instance for both types exist, and is similar. You may just `put s` directly there.